report/test/renderer/report-ui-features-test.js

```diff
@@ -111,12 +111,44 @@ describe('ReportUIFeatures', () => {
         const webpAuditItemTemplate = {
           ...sampleResults.audits['modern-image-formats'].details.items[0],
           wastedBytes: 8.8 * 1024,
+          entity: undefined, // Remove entity classification from previous result.
         };
+
         const renderBlockingAuditItemTemplate =
           sampleResults.audits['render-blocking-resources'].details.items[0];
         const textCompressionAuditItemTemplate =
           sampleResults.audits['uses-text-compression'].details.items[0];
 
+        // Setup entity-classification with recognized entities first.
+        lhr.entityClassification = {
```

Comment:
We should still have a test making sure this continues to work for back compat (LHRs w/o entityClassification)

Reply:
I reversed [this test](https://github.com/GoogleChrome/lighthouse/pull/14614/files#diff-1acce8202de9589b2148f5e8948fa0182fa895a52575ae2ceca54d2b2ce31f90R334) (and a few other) during the branch creation reverse-patch. I'll add this one back, good catch!

Reply:
Added back the legacy tests, and refactored with an outer level `describe()` to contain both new and legacy.